js/src/components/SEMrushRelatedKeyphrasesModal.js

```diff
@@ -55,38 +56,80 @@ class SEMrushRelatedKeyphrasesModal extends Component {
 		this.props.onClose();
 	}
 
+	/**
+	 * Opens the popup window.
+	 *
+	 * @param {event} e The click event.
+	 *
+	 * @returns {void}
+	 */
+	onLinkClick( e ) {
+		const url = "https://oauth.semrush.com/oauth2/authorize?" +
+			"ref=1513012826&client_id=yoast&redirect_uri=https%3A%2F%2Foauth.semrush.com%2Foauth2%2Fyoast%2Fsuccess&response_type=code&scope=user.id";
+		const height = "570";
+		const width  = "340";
+		const top = window.top.outerHeight / 2 + window.top.screenY - ( height / 2 );
+		const left = window.top.outerWidth / 2 + window.top.screenX - ( width / 2 );
```

Comment:
According to MDN, `outerHeight` and `outerWidth` are browser-dependant and shouldn't be used: https://developer.mozilla.org/en-US/docs/Web/API/Window/open. Is this safe to use or maybe we should use better methods?

Reply:
Those seem to be part of the `windowFeatures` passed as a parameter to the `window.open()`.
The above code instead reads the values of the properties from the `window` object, and it seems that they are completely standard on every browser since ages :-)
https://developer.mozilla.org/en-US/docs/Web/API/Window/outerHeight
https://developer.mozilla.org/en-US/docs/Web/API/Window/outerWidth

Reply:
Thanks for the clarification 😄